tests/testnameresolve.nim

```
@@ -56,8 +57,10 @@ proc guessOsNameServers(): seq[TransportAddress] =
           if resultSeq.len > 2:
             break
             #3 nameserver max on linux
-    except CatchableError as err:
-      echo "Failed to get unix nameservers ", err.msg
+    except IOError as exc:
+      debug "Failed to get unix nameservers", description = exc.msg
+    except TransportAddressError as exc:
+      debug "Failed to init address", description = exc.msg
```

could `raiseAssert` be used here instead of fancy echo - log?